mne/io/edf/edf.py

```diff
@@ -473,6 +474,13 @@ def _get_info(fname, stim_channel, eog, misc, exclude, preload):
     if np.isnan(info['lowpass']):
         info['lowpass'] = info['sfreq'] / 2.
 
+    if info['highpass'] > info['lowpass']:
+        warn(f'Highpass cutoff frequency {info["highpass"]} is greater than '
+             f'lowpass cutoff frequency {info["lowpass"]}. '
+             'Setting both values to None.')
+        info['highpass'] = None
+        info['lowpass'] = None
```

Comment:
Or should we set them to `0` and `info['sfreq'] / 2` instead? I guess this amounts to the same thing (where do we equal `None` to these values?), but it would be more explicit (maybe at least in the warning message).